hedera-node/hedera-app/src/main/java/com/hedera/node/app/records/schemas/V0490BlockRecordSchema.java

```diff
@@ -67,7 +67,8 @@ public void migrate(@NonNull final MigrationContext ctx) {
         final var isGenesis = ctx.previousVersion() == null;
         if (isGenesis) {
             final var blocksState = ctx.newStates().getSingleton(BLOCK_INFO_STATE_KEY);
-            final var blocks = new BlockInfo(-1, EPOCH, Bytes.EMPTY, EPOCH, false, EPOCH);
+            // Note there is by convention no post-upgrade work to do if starting from genesis
+            final var blocks = new BlockInfo(-1, EPOCH, Bytes.EMPTY, EPOCH, true, EPOCH);
```

Comment:
This fix also I assume is just better, but didn't actually contribute to the fix of the bug that triggered this PR? (It is great to include this, I just want to make sure I understand what is happening)